CHANGELOG.md

```diff
@@ -25,6 +25,18 @@ Please update your configuration accordingly.
 If you don't update the `actor-factory` value, you'll get a deprecation warning in the logs, and Cromwell will default back to
 **PAPI V1**
 
+**Requester Pays**
+
+Cromwell now supports [Requester Pays](https://cloud.google.com/storage/docs/requester-pays) feature for Cloud Storage. With Requester Pays enabled on your bucket,
+you can require requesters to include a billing project in their requests, thus billing the requester's project. If the bucket you are accessing has requester pays enabled,
+to access the file inside that bucket you need to mention the Google project id which can be billed for that request. The billing project information can be added in ways described below. Cromwell will
+look for this information in this order.
+1. It can be added as `'google_project':'project-id'` as part of workflow options during workflow submission
+2. It can be included inside configuration file as shown in [`Getting started on Google Pipelines API`](http://cromwell.readthedocs.io/en/develop/tutorials/PipelinesApi101/) where you need to replace the `<google-billing-project-id>` with the project id **(HIGHLY RECOMMENDED)**
```

Comment:
Please put the technical details in the README or ReadTheDocs documentation. 

IMO the change log is best is you can read through the entries in it quickly and it links out to longer details on all the topics that you care about.